config/locales/views/submissions/en.yml

```diff
@@ -41,6 +41,7 @@ en:
       no_submission: "%{group_name} has no submissions"
       not_complete: 'Cannot release result for %{group_name}: the marking is not complete.'
       not_complete_unrelease: 'Cannot unrelease result for %{group_name}: the marking is not complete.'
+    file_extension_mismatch: The contents of the file do not match the file extension.
```

Comment:
Let's be more specific here, since I'd like this to display for any failed JSON parsing when converting notebooks. An appropriate error is something like "Invalid Jupyter notebook contents."